### PR TITLE
removing the vfp_table member from Group class

### PR DESCRIPTION
**Pull request by GitPaean**
after GRUPNET is processed, vfp_table is not useful for GROUP class anymore.

### CODE REVIEW

**Comment by GitPaean** [comment]
jenkins build this please

**Comment by GitPaean** [comment]
jenkins build this please

File: python/cxx/group.cpp [hunk 1]
```diff
@@ -8,18 +8,13 @@ namespace {
...
-    }
 }
 
 void python::common::export_Group(py::module& module) {
 
   py::class_< Group >( module, "Group")
     .def_property_readonly( "name", &Group::name)
-    .def_property_readonly( "num_wells", &Group::numWells)     
-    .def( "_vfp_table_nr", &get_vfp_table_nr )
+    .def_property_readonly( "num_wells", &Group::numWells)
```
Comment (GitPaean) [inline python/cxx/group.cpp:17]: I could not see what is changed in this line.

Comment (GitPaean) [reply python/cxx/group.cpp:17]: That is also what I thought. But I thought there might be a light color to show the difference.

Comment (akva2) [reply python/cxx/group.cpp:17]: there is not, but you can select with the mouse to see it.

**Review by akva2 — APPROVED** [review]
All good.

**Comment by GitPaean** [comment]
With the approval, I am getting this in now. It is something pretty clear and should not be controversial.

**Pull request merged**